src/main/java/org/jenkinsci/plugins/ewm/steps/model/ExternalWorkspace.java

```diff
@@ -2,27 +2,35 @@
 
 import javax.annotation.Nonnull;
 import java.io.Serializable;
+import java.util.UUID;
 
 /**
  * POJO used to pass fields from one step to another.
  *
  * @author Alexandru Somai
  */
-public final class ExternalWorkspace implements Serializable {
+public class ExternalWorkspace implements Serializable {
 
     private static final long serialVersionUID = 1L;
 
+    private final String id;
```

Comment:
NIT for now, but in after 1.0 such changes will require readResolve() for data migration